src/core_plugins/metrics/server/routes/fields.js

```diff
@@ -7,7 +7,12 @@ export default (server) => {
     handler: (req, reply) => {
       return getFields(req)
         .then(reply)
-        .catch(() => reply([]));
+        .catch((err) => {
+          if (err.isBoom) {
```

Comment:
Is there a better condition that we could use here? The fact that the error is a Boom error doesn't really mean anything specific... Maybe a check for `err.output.statusCode === 401`?